bindings/python/bipedal_locomotion.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file bipedal_locomotion.cpp
+ * @authors Giulio Romualdi, Diego Ferigo
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <pybind11/pybind11.h>
+
+#include "BipedalLocomotion/Planners/Contact.h"
+#include "bipedal_locomotion.h"
```

Comment:
Probably both ``bipedal_locomotion.h`` and ``bipedal_locomotion.cpp`` should be renamed to ``bipedal_locomotion_framework`` right?

Reply:
Right, done in 4f60bde, this time squashing the change was too tricky, I'll keep the commit.